app.py

```diff
@@ -137,6 +137,15 @@ def upload():
             file.save(destination)
             os.system('python utils/encryption.py')
             flash("File Uploaded", "success")
+             //this is a temp hardcoded value for node we will change it later. 
+            test_response = requests.post( 'http://localhost:3000/upload' , files = {"form_field_name": file})
```

Comment:
Correct me if im wrong,
the place where localhost link is given, if we parse other nodes ip's , would the data be post across them all?

Reply:
yes i would work .